api/resources/reports.py

```diff
@@ -0,0 +1,138 @@
+import datetime
+import json
+
+import bleach
+from flask import request
+from flask_restful import Resource, abort
+from sqlalchemy.orm.exc import NoResultFound
+
+from api import db
+from api.database.models import Report
+
+
+def _validate_field(data, field, proceed, errors, missing_okay=False):
+    if field in data:
+        # sanitize the report input here
+        data[field] = bleach.clean(data[field].strip())
+        if len(data[field]) == 0:
```

Comment:
@PhilipDeFraties I think we ended up deleting line 15 and editing line 17 like so `if len(str(data[field])) == 0:`...python doesn't sanitize floats and won't check their lengths...so we just changed it to a string class IOT work...